setup.py

```diff
@@ -18,4 +19,19 @@
                                          "config/fedora.json",
                                          "config/redhat.json"])],
     license='GPLv3+',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
+        'Operating System :: POSIX :: Linux',
+        'Programming Language :: Python :: 3',
+        'Topic :: Software Development',
+        'Topic :: Software Development :: Quality Assurance',
+        'Topic :: Utilities',
+    ],
+    keywords='containers,sanity',
+    author='Red Hat',
+    author_email='user-cont@redhat.com',
```

Comment:
This is internal email address. Assign it to yourself.

Reply:
I've copied this from conu..;-)